.github/workflows/scale-test-100-gce.yaml

```diff
@@ -0,0 +1,222 @@
+name: 100 Nodes scale test
```

Comment:
For consistency, can we make it title 'Nodes Scale Test'
Also a question, it seems a very spesific name '100 Nodes Scale Test', will there be other tests like 250?

Reply:
"100" is on purpose here. The idea is that we can compare easily 1:1 the results of this scale test with [k8s scale test](https://testgrid.k8s.io/sig-scalability-gce#gce-cos-master-scalability-100) to see in what areas we under/overperform. Next one would be 5k nodes :)